### PR TITLE
APPSREPO-152: Added Password Reset API spec.

### PR DESCRIPTION
**Pull request by jamalkm**
@gavincornwell please review.
Thanks

### CODE REVIEW

File: src/main/webapp/definitions/alfresco-core.yaml [hunk 1]
```diff
@@ -7347,4 +7430,22 @@ definitions:
           - DONE
           - MAX_CONTENT_SIZE_EXCEEDED
         default: PENDING
-        description: the current status of the download node creation
\ No newline at end of file
+        description: the current status of the download node creation
+  ClientBody:
+    type: object
```
Comment (gavincornwell) [inline src/main/webapp/definitions/alfresco-core.yaml]: missing required declaration? I would have thought all properties in the body were mandatory.

File: src/main/webapp/definitions/alfresco-core.yaml [hunk 2]
```diff
@@ -7347,4 +7430,22 @@ definitions:
...
+    properties:
+      client:
+        type: string
+        description: the client name
+  PasswordResetBody:
+    type: object
```
Comment (gavincornwell) [inline src/main/webapp/definitions/alfresco-core.yaml]: missing required declaration? I would have thought all properties in the body were mandatory.

**Comment by jamalkm** [comment]
Thanks @gavincornwell. Added the missing required declaration.

File: src/main/webapp/definitions/alfresco-core.yaml (hunk 2 above)
Comment (jamalkm) [inline src/main/webapp/definitions/alfresco-core.yaml]: Added the missing required declaration.

**Comment by gavincornwell** [comment]
Feel free to merge when you're ready.

**Pull request merged**